src/blocks/TextEditBlock.tsx

```diff
@@ -38,24 +38,26 @@ export default class TextEditBlock extends Block {
         onChange={onEditorStateChange}
         blockStyleFn={this.blockStyleFn}
         readOnly={true}
-      />);
+      />
+    );
   }
 
   renderEditModal(done: (data: BlockModel) => void) {
-    let textState = this.model.data["textState"]
+    let textState = this.model.data["textState"];
     return (
-      <div className="Editor" id={this.model.uuid}>
```

Comment:
this removes the css styling from `TextEditorStyles.css` which is fine, but it should be cleaned up if we aren't planning to use those classes anymore